components/brave_wallet_ui/components/extension/confirm-transaction-panel/common/evm_state_changes.stories.tsx

```diff
@@ -35,118 +35,136 @@ function assertDefined<T>(data: T) {
-          mockSendEthEvent.rawInfo.data.nativeAssetTransferData
+export const _Erc1155TokenTransfer = {
+  render: () => {
+    return (
+      <WalletPanelStory>
+        <NonFungibleErcTokenTransfer
+          network={mockEthMainnet}
+          transfer={assertDefined(
+            mockReceiveMultiStandardTokenEvent.rawInfo.data.erc1155TransferData
+          )}
+        />
+      </WalletPanelStory>
+    )
+  }
+}
+
+export const _Erc20TokenApproval = {
+  render: () => {
+    return (
+      <ErcTokenApproval
+        approval={assertDefined(
+          mockApproveUsdtEvent.rawInfo.data.erc20ApprovalData
         )}
         network={mockEthMainnet}
+        isApprovalForAll={false}
+        isERC20
       />
-    </WalletPanelStory>
-  )
+    )
+  }
 }
 
-export const _Erc721TokenTransfer = () => {
-  return (
-    <WalletPanelStory>
-      <NonFungibleErcTokenTransfer
-        network={mockEthMainnet}
-        transfer={assertDefined(
-          mockReceiveNftEvent.rawInfo.data.erc721TransferData
+export const _Erc721TokenApproval = {
+  render: () => {
+    return (
+      <ErcTokenApproval
+        approval={assertDefined(
+          mockApproveBoredApeNftTransferEvent.rawInfo.data.erc721ApprovalData
         )}
+        network={mockEthMainnet}
+        isApprovalForAll={false}
+        isERC20={false}
       />
-    </WalletPanelStory>
-  )
+    )
+  }
 }
 
-export const _Erc1155TokenTransfer = () => {
-  return (
-    <WalletPanelStory>
-      <NonFungibleErcTokenTransfer
-        network={mockEthMainnet}
-        transfer={assertDefined(
-          mockReceiveMultiStandardTokenEvent.rawInfo.data.erc1155TransferData
+export const _Erc721TokenApprovalForAll = {
+  render: () => {
+    return (
+      <ErcTokenApproval
+        approval={assertDefined(
+          mockApproveAllBoredApeNFTsEvent.rawInfo.data.erc721ApprovalForAllData
         )}
+        network={mockEthMainnet}
+        isApprovalForAll={true}
+        isERC20={false}
       />
-    </WalletPanelStory>
-  )
-}
-
-export const _Erc20TokenApproval = () => {
-  return (
-    <ErcTokenApproval
-      approval={assertDefined(
-        mockApproveUsdtEvent.rawInfo.data.erc20ApprovalData
-      )}
-      network={mockEthMainnet}
-      isApprovalForAll={false}
-      isERC20
-    />
-  )
-}
-
-export const _Erc721TokenApproval = () => {
-  return (
-    <ErcTokenApproval
-      approval={assertDefined(
-        mockApproveBoredApeNftTransferEvent.rawInfo.data.erc721ApprovalData
-      )}
-      network={mockEthMainnet}
-      isApprovalForAll={false}
-      isERC20={false}
-    />
-  )
+    )
+  }
 }
 
-export const _Erc721TokenApprovalForAll = () => {
-  return (
-    <ErcTokenApproval
-      approval={assertDefined(
-        mockApproveAllBoredApeNFTsEvent.rawInfo.data.erc721ApprovalForAllData
-      )}
-      network={mockEthMainnet}
-      isApprovalForAll={true}
-      isERC20={false}
-    />
-  )
+export default {
+  component: ErcTokenApproval
```

Comment:
Sweet :+1: It's a bit odd because Storybook takes the title of the component and in my (limited) testing you need to specify either a component or a title (which is used for the group). Messed up this one :laughing: